### PR TITLE
Add support to run tests with sauce credentials Vslt/vslt 1243

### PR DESCRIPTION
**Pull request by luisVelasquezSauce**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## 0.20.4
```
Comment (loyalchow) [inline CHANGELOG.md:3]: Update minor version

File: test/unit/validate.spec.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 var expect = require('chai').expect;
 var validate = require('../../src/validate');
 
-describe('screener-storybook/src/validate', function() {
+describe.only('screener-storybook/src/validate', function() {
```
Comment (loyalchow) [inline test/unit/validate.spec.js:4]: Do you want this `only`?

File: test/unit/validate.spec.js [hunk 3]
```diff
@@ -10,10 +10,17 @@ describe('screener-storybook/src/validate', function() {
         });
     });
 
-    it('should throw error when no apiKey', function() {
-      return validate.storybookConfig({})
+    it('should throw error when no apiKey or username', function() {
```
Comment (loyalchow) [inline test/unit/validate.spec.js:13]: Can you add a test for the xor? That both apiKey and username/accessKey can't exist at the same time

**Review by loyalchow — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (luisVelasquezSauce) [reply CHANGELOG.md]: Done

File: test/unit/validate.spec.js (hunk 2 above)
Comment (luisVelasquezSauce) [reply test/unit/validate.spec.js]: Removed

File: test/unit/validate.spec.js (hunk 3 above)
Comment (luisVelasquezSauce) [reply test/unit/validate.spec.js:13]: Added!

**Comment by loyalchow** [comment]
@luisVelasquezSauce LGTM 👍 - as discussed, please don't merge yet till App updates are deployed

**Review by loyalchow — APPROVED** [review]

**Pull request closed without merging**